appeals/web/src/server/appeals/appeal-details/status-tags/status-tags.mapper.js

```diff
@@ -0,0 +1,151 @@
+import { mapVirusCheckStatus } from '#appeals/appeal-documents/appeal-documents.mapper.js';
+import { dateISOStringToDisplayDate, getTodaysISOString } from '#lib/dates.js';
+import logger from '#lib/logger.js';
+import { generateDecisionDocumentDownloadHtml } from '#lib/mappers/appeal/appeal.mapper.js';
+import { APPEAL_CASE_STATUS, APPEAL_VIRUS_CHECK_STATUS } from 'pins-data-model';
+import { getAppealTypesFromId } from '../change-appeal-type/change-appeal-type.service.js';
+
+/**
+ * @param {{ appeal: MappedInstructions }} mappedData
+ * @param {import('../appeal-details.types.js').WebAppeal} appealDetails
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @returns {Promise<PageComponent[]>}
+ */
+export const generateStatusTags = async (mappedData, appealDetails, request) => {
+	/** @type {PageComponent|undefined} */
+	let statusTag;
+
+	if (mappedData.appeal.appealStatus.display?.statusTag) {
```

Comment:
Would it be worth breaking down the Status tags by type while you are at it? Would probably make reading this a little easier